src/main/java/com/ase/restservice/service/TransactionService.java

```diff
@@ -111,4 +112,8 @@ public Optional<Asset> sellTransaction(Transaction transaction, Stock stock) thr
     updateTransactionStatus(transaction, "COMPLETED");
     return newAsset;
   }
+
+  public List<Transaction> getAccountTransactions(String accountId) {
```

Comment:
We also need a listTransactions method and its endpoint for returning all transactions. This was asked by our TA